.ci_support/linux_.yaml

```diff
@@ -4,6 +4,10 @@ cxx_compiler:
 - toolchain_cxx
 icu:
 - '58'
+perl:
```

Comment:
@CJ-Wright is this correct? `erl`?

Reply:
What do you mean?

Reply:
I see `erl` and `cu` instead of `perl` and `icu`.

Reply:
Actually, when I click on the file, everything is fine! I guess some of my chrome extensions are at fault here:

![screenshot from 2018-07-26 19-38-28](https://user-images.githubusercontent.com/950575/43292439-93b2a7e0-910b-11e8-925a-2e6f1af4e5cc.png)

Reply:
ooh that's interesting

Reply:
It was https://chrome.google.com/webstore/detail/refined-github/hlepfoohegkhhmjieoechaddaejaokhf BTW in case you are curious.

(Sorry for the noise!)